IntelliTect.TestTools.Data/Log.cs

```diff
@@ -35,7 +35,7 @@ public class Log
         public Log(
             LogLevel logLevel,
             EventId eventId,
-            object? state,
+            object state,
```

Comment:
I would add a null check on the reference types since this class is public.

Reply:
I dont think Log implementations are guaranteed to provide a state. That being said I don't know if this is worthy of public. Do you think I should just make the constructor internal?